examples/active.ini

```diff
@@ -0,0 +1,46 @@
+[acq_params]
+adc_offset = 30
+amplitude = 1.0
+p90_us = 4.8
+deadtime_us = 10.0
+tau_us = 3500.0
+deblank_us = 1.0
+sw_khz = 3.9
+acq_time_ms = 1024.0
+nscans = 1
+thermal_nscans = 2
+carrierfreq_mhz = 14.895
+nechoes = 1
+startconstant = 0.15
+stopconstant = 2.0
+gamma_eff_mhz_g = 0.004236890737777777
+mw_freqs = 9820774000.0
+field_width = 8
```

Comment:
used to make field sweep axis. 8 means 8 points will be collected with 1.0 as spacing. 1.0 is hardset in the run_field_dep_mw.py or run_FS.py